test/simple-concat-test.js

```diff
@@ -28,74 +29,69 @@ describe('simple-concat', function() {
     }
   });
 
-  it('does not generate sourcemaps', function() {
+  it('does not generate sourcemaps', co.wrap(function *() {
     var node = concat(firstFixture, {
       header: "/* This is my header. */",
       inputFiles: ['**/*.js'],
       outputFile: '/no-sourcemap.js',
       sourceMapConfig: { enabled: false },
     });
     builder = new broccoli.Builder(node);
-    return builder.build().then(function(result) {
-      expectFile('no-sourcemap.js').in(result);
-      expectFile('no-sourcemap.map').notIn(result);
-    });
-  });
+    let result = yield builder.build();
```

Comment:
If you are going to use `let` here you need to add `”use strict”;` to the top (this is what is failing node 4)